### DISCUSSION TITLE
does it support m1 mac ?

### DISCUSSION THREAD

**Discussion by rickstar666** [Zulko/moviepy#1839] [Q&A]
i successed import and run it in my windows pc.
my firends failed. it always say 

RuntimeError: No ffmpeg exe could be found. Install ffmpeg on your system, or set the IMAGEIO_FFMPEG_EXE environment variable.

is it the result of the m1 mac ?

**Comment by keikoro** [comment]
Please recreate this as an issue (there's a template for questions), thank you.
